app/api/connectors/httpparsers/MtdIdLookupHttpParser.scala

```diff
@@ -17,9 +17,10 @@
 package api.connectors.httpparsers
 
 import api.connectors.MtdIdLookupOutcome
-import api.models.errors.{InternalError, InvalidBearerTokenError, NinoFormatError}
 import play.api.http.Status.{FORBIDDEN, OK, UNAUTHORIZED}
 import play.api.libs.json.{Reads, __}
+import shared.models.errors
+import shared.models.errors.{InvalidBearerTokenError, NinoFormatError}
```

Comment:
Is there a reason we have specific imports for `InvalidBearerTokenError` and `NinoFormatError` but not `InternalError`? 
```suggestion
import shared.models.errors.{InvalidBearerTokenError, NinoFormatError, InternalError}
```